src/DOMupdates.js

```diff
@@ -37,9 +34,9 @@ let domUpdates = {
     let findClues = clues.cards;
     findClues.forEach((clue, index) => {
       $(".card").on("click", function(e) {
-        $(e.target).closest($(`#js-row-${index}`)).html(`<p>${clue.question}</p>`
-        )
-      })
+      $(e.target).closest($(`#js-row-${index}`)).html(`<h3>${clue.question}</h3>
```

Comment:
Hey @yaleksandrova, Please change back to the following. This will end up breaking the functionality for displaying the questions:    
$(e.target).closest($(`#js-row-${index}`)).html(`<p>${clue.question}</p>`)
 })